pkg/placement/membership.go

```diff
@@ -239,8 +267,14 @@ func (p *Service) disseminateOperation(targets []placementGRPCStream, operation
 	var err error
 	for _, s := range targets {
 		err = s.Send(o)
+
 		if err != nil {
-			log.Errorf("error updating host on unlock operation: %s", err)
+			peerAddr := "n/a"
+			if peer, ok := peer.FromContext(s.Context()); ok {
```

Comment:
please add more comments here.

Reply:
this is to get the remote client address. I renamed the variable name instead of adding comment. code is clear so that I do not think i need to add additional comment here.